### PR TITLE
Make AppDeployIntegrationTest less shaky

### PR DESCRIPTION
**Pull request by aquamatthias**

### CODE REVIEW

File: src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala [hunk 1]
```diff
@@ -314,22 +313,18 @@ class AppDeployIntegrationTest
 
   test("kill all tasks of an App with scaling") {
     Given("a new app with multiple tasks")
```
Comment (gkleiman) [inline src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala]: If you change the instances count, then you also have to change this description ;-).

File: src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala [hunk 2]
```diff
@@ -314,22 +313,18 @@ class AppDeployIntegrationTest
...
-    val app = v2AppProxy(testBasePath / "tokill", "v1", instances = 2, withHealth = false)
+    val app = v2AppProxy(testBasePath / "tokill", "v1", instances = 1, withHealth = false)
```
Comment (gkleiman) [inline src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala]: If you have only 1 instance, how can you be sure that killAllTasks will kill > 1 task?

Comment (kolloch) [inline src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala]: And we DID have errors like that in the past. I also would like it if you kept the count at 2.

Comment (aquamatthias) [inline src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala]: The only reason I switched to 1 is: to be able to identify the kill event __and__ the deployment success. Reverted and removed the waitForEvents method.

**Pull request merged**